.travis.yml

```diff
@@ -8,7 +8,9 @@ python:
 install:
     - sudo apt-get install graphviz || sudo apt-get update && sudo apt-get install graphviz
     - pip install coverage coveralls
+    - pip install flake8
     - pip install -e .
+before_script: "make lint"
```

Comment:
I'd prefer

```
before_script:
    - make lint
```

Also, I think I'd prefer to `pip install coverage coveralls flake8` on a single line.  Maybe even `pip install coverage coveralls flake8 -e .`, if that doesn't look too obfuscated?  Or maybe `pip install -r requirements.txt` and a `reuiqrements.txt` that lists them all, including `-e .`?

I hate it when I start bikeshedding with myself, but I can't help it :(